<launch>
    <node pkg="prometheus_detection" type="kcf_tracker" name="kcf_tracker" output="screen">
        <param name="camera_topic" type="string" value="/amov_solo_monocular/Monocular/image_raw" />
        <param name="camera_info" type="string" value="$(find prometheus_experiment)/config/prometheus_detection_config/camera_param_gazebo_monocular.yaml" />
    </node>
</launch>
